docs/entities/kpis.md

```diff
@@ -34,17 +34,30 @@ For more info on [asset_selection](/docs/commonly_used_data_types/asset_selectio
 In order to make use of the data viz function you'll need the name of the KPI you wish to get.
 
 ### Get KPI Data Viz
-Once you have the name of the KPI you wish to access and a logged in client you can make a call to the data viz api with the following SDK function call:
+
+The `get_kpi_data_viz` function allows you to access KPI data through the data viz API. The SDK function call has undergone a signature update without changing the internal implementation. You can use any one of the following SDK function calls:
+
+#### Old API:
+
 ```
 cli.get_kpi_data_viz(machine_source, kpis, i_vars, time_selection, **optional_data_viz_query)
 ```
 
-After some time the SDK should return a list that looks something like this:
+#### New API:
+
+```
+cli.get_kpi_data_viz(*args, **kwargs)
+```
```

Comment:
make **kwargs **data_viz_query this makes for a much clearer reading exprience

Reply:
Done.